enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/CleanupCleanState.tsx

```diff
@@ -0,0 +1,15 @@
+import { t } from "ttag";
+
+import { CollectionEmptyIcon } from "metabase/collections/components/CollectionEmptyState/CollectionEmptyState";
+import { Flex, Text } from "metabase/ui";
+
+export const CleanupCleanState = ({ duration }: { duration: string }) => {
+  return (
+    <Flex align="center" direction="column" pt="10rem" pb="11rem">
+      <CollectionEmptyIcon />
+      <Text fw="bold" size="1.25rem" mt="4rem">
+        {t`We didn’t find any items not viewed in over ${duration}`}
```

Comment:
This is a double negative so it might confuse - worth another look?

Maybe something like this could work - "We didn't find any items neglected for ${duration}"

Reply:
Or something like "We couldn't find any items last viewed more than two months ago"

Btw is it a _view_ or an _edit_ that makes a difference? When I deleted an item then restored it, I think it no longer appeared in the clean-up modal because it was considered recently edited (though it was not recently viewed).

Reply:
Another thought about the text: since the dropdown already makes clear what kinds of things are supposed to appear in the modal, perhaps "No items found" is enough?

Reply:
How about `All items have been used in the past 2 months`?

I think we were using "view" in the past and this needs to be "used". Used can mean a lot of things (view, edited, run, etc).